recipes/jp-noui/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: jp-noui
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/jp_noui/jp_noui-{{ version }}.tar.gz
+  sha256: 4e1e8ba5c9aef02432559d234c751bddccfbf72ede3902ef7cd2dc9fcac50a1c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+
+requirements:
+  host:
+    - hatch-jupyter-builder
+    - hatch-nodejs-version
+    - jupyterlab >=3,<4
+    - nodejs
+    - pip
+    - python >=3.7
+
+  run:
+    - python >=3.7
+
+  run_constrained:
+    - jupyterlab >=3,<4
+
+test:
+  requires:
+    - pip
+    - jupyterlab >=3,<4
+  imports:
+    - jp_noui
+  commands:
+    - test -f ${PREFIX}/share/jupyter/labextensions/jp-noui/package.json                            # [unix]
+    - if not exist %PREFIX%\\share\\jupyter\\labextensions\\jp-noui\\package.json (exit 1)          # [win]
+    - jupyter labextension list
+    - pip check
+
+about:
+  home: https://github.com/JoelStansbury/jp-noui
+  summary: Hides jupyterlab UI elements for presentable notebook applications
+  license: BSD-3-Clause
+  license_file:
+    - LICENSE
```

Comment:
recommend including the nodejs-side `third-party-licenses.json` e.g.

https://github.com/conda-forge/ipyforcegraph-feedstock/blob/main/recipe/meta.yaml#L51

Reply:
That file does not exist for me. Is that json something you needed to create or does is that supposed to be created automatically?

Reply:
[5dcec59](https://github.com/conda-forge/staged-recipes/pull/22083/commits/5dcec59e70f6aab4e33cf2de0814cd8a675b96b6)